L1Trigger/L1TCaloLayer1/plugins/L1TCaloSummary.cc

```diff
@@ -184,6 +180,11 @@ void L1TCaloSummary<INPUT, OUTPUT>::produce(edm::Event& iEvent, const edm::Event
 
   UCTGeometry g;
 
+  hls4mlEmulator::ModelLoader loader(modelName);
+  std::shared_ptr<hls4mlEmulator::Model> model;
+  //anomaly trigger loading
+  model = loader.load_model();
```

Comment:
what is the impact of the job timing / throughput of creating an `hls4mlEmulator::ModelLoader` object and loading the model for every event ?